packages/frontend/src/components/EpochKeyEvent.jsx

```diff
@@ -1,41 +1,21 @@
-import React, { useContext, useEffect } from 'react'
+import React from 'react'
 import { observer } from 'mobx-react-lite'
-import state from '../contexts/state'
 import dayjs from 'dayjs'
 import relativeTime from 'dayjs/plugin/relativeTime'
 import './eventCard.css'
 
 dayjs.extend(relativeTime)
 
 export default observer(({ attestation }) => {
-  const { attesterId, epoch, posRep, negRep, graffiti, timestamp } = attestation
-  const { attester } = useContext(state)
-  useEffect(() => {
-    const loadData = async () => {
-      await attester.loadEpochsByAttester(attesterId)
-    }
-    loadData()
-  }, [])
-  const attesterEpoch = attester.epochsByNumber.get(epoch)
```

Comment:
moved this to new EpochKeyInfo component in order to get `epochSealed?` into the info box on the left.